aptible-cli.gemspec

```diff
@@ -25,9 +25,10 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'aptible-auth', '~> 1.2.3'
   spec.add_dependency 'aptible-billing', '~> 1.0'
   spec.add_dependency 'thor', '~> 0.20.0'
-  spec.add_dependency 'git'
+  spec.add_dependency 'git', '< 1.10'
   spec.add_dependency 'term-ansicolor'
   spec.add_dependency 'chronic_duration', '~> 0.10.6'
+  spec.add_dependency 'cbor'
```

Comment:
The authenticator data the is provided to use by `fido2-assert` is encoded with CBOR so we need to decode it before sending it to `auth-api`